client/app/private-database/database/add/private-database-database-add.html

```diff
@@ -119,6 +119,41 @@
                     </div>
                 </div>
             </form>
+
+            <form class="form-horizontal" name="privateDatabaseAddAuthorizedIpForm" ng-if="isDBaaS() && !addBddCtrl.hasAuthorizedIp">
```

Comment:
This function isn't linked to `addBddCtrl` ?

Reply:
It's in the parent's $scope :/